pkg/authorization/apis/authorization/conversion.go

```diff
@@ -154,6 +155,7 @@ func getRBACRoleRefKind(namespace string) string {
 
 func Convert_rbac_ClusterRole_To_authorization_ClusterRole(in *rbac.ClusterRole, out *ClusterRole, _ conversion.Scope) error {
 	out.ObjectMeta = in.ObjectMeta
+	translate_rbac_Annotations_To_Origin_Annotation(out)
```

Comment:
There should be unit tests for this in both directions (just like there is one for every other special case conversion).